front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -288,37 +337,51 @@ const ScenarioV2 = () => {
               )}
               <div className="scenario-results">
                 {collapsedTimetable && (
-                  <div className="scenario-timetable-collapsed">
-                    <button
-                      className="timetable-collapse-button"
-                      type="button"
-                      aria-label={t('toggleTimetable')}
-                      onClick={() => setCollapsedTimetable(false)}
-                    >
-                      <ChevronRight />
-                    </button>
-                    <div className="lead ml-2">{scenario.name}</div>
-                    <div className="d-flex align-items-center ml-auto">
-                      <img src={infraLogo} alt="Infra logo" className="infra-logo mr-2" />
-                      {scenario.infra_name}
-                    </div>
-                    <div className="d-flex align-items-center ml-4">
-                      <span className="mr-1">
-                        <GiElectric />
-                      </span>
-                      {timetable.electrical_profile_set_id
-                        ? timetable.electrical_profile_set_id
-                        : t('noElectricalProfileSet')}
+                  <>
+                    <div className="scenario-timetable-collapsed">
+                      <button
+                        className="timetable-collapse-button"
+                        type="button"
+                        aria-label={t('toggleTimetable')}
+                        onClick={() => setCollapsedTimetable(false)}
+                      >
+                        <ChevronRight />
+                      </button>
+                      <div className="lead ml-2">{scenario.name}</div>
+                      <div className="d-flex align-items-center ml-auto">
+                        <img src={infraLogo} alt="Infra logo" className="infra-logo mr-2" />
+                        {scenario.infra_name}
+                      </div>
+                      <div className="d-flex align-items-center ml-4">
+                        <span className="mr-1">
+                          <GiElectric />
+                        </span>
+                        {timetable.electrical_profile_set_id
+                          ? timetable.electrical_profile_set_id
+                          : t('noElectricalProfileSet')}
+                      </div>
                     </div>
-                  </div>
+                    <MicroMacroButtons
+                      isMacro={isMacro}
+                      handleClickButton={handleClickMicroMacroButton}
+                      t={t}
+                    />
```

Comment:
Since we're using this at multiple spots, maybe we can isolate it in a `const`?

Reply:
Done

Reply:
Thank you for your review